src/index.ts

```diff
@@ -345,6 +350,37 @@ class Spanner extends GrpcService {
       [CLOUD_RESOURCE_HEADER]: this.projectFormattedName_,
     };
     this.directedReadOptions = directedReadOptions;
+    this._instance_admin_api = null;
+    this._database_admin_api = null;
```

Comment:
nit: Isn't this redundant, as they are already set to null when they are defined?

Reply:
Updated